ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -180,12 +165,8 @@ const mapStateToProps = (state: GlobalState): ReduxMappedStateProps => ({
   cards: walletsSelector(state)
 });
 
-const mapDispatchToProps = (dispatch: Dispatch): ReduxMappedDispatchProps => ({
-  loadTransactions: () => dispatch(fetchTransactionsRequest()),
-  loadWallets: () => dispatch(fetchWalletsRequest())
-});
-
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(WalletHomeScreen);
+export default withLoadingSpinner(
```

Comment:
mmh I think this approach sucks, we can't have a root screen with a loading spinner, it doesn't make much sense that I open the app, I go to the wallet tab and I see a loading spinner, I can't even scan a QR code! 

I suggest you deal with the loading state of the payment methods and transactions separately from the wallet screen.

Reply:
Agreed, this is going to take me some more time to finish then. I didn't think about it earlier, but it does make sense to have 2 separate spinners, one for the cards and one for the transactions, that are only shown when the respective data is being fetched